src/app/componentRegistry.js

```diff
@@ -8,6 +8,7 @@ import CellSetsManagerSubscriber from '../components/sets/CellSetsManagerSubscri
 import HoverableScatterplotSubscriber from '../components/scatterplot/HoverableScatterplotSubscriber';
 import HoverableSpatialSubscriber from '../components/spatial/HoverableSpatialSubscriber';
 import HoverableHeatmapSubscriber from '../components/heatmap/HoverableHeatmapSubscriber';
+import ChannelEffectSubscriber from '../components/channel-effects/ChannelEffectSubscriber';
```

Comment:
Maybe:
```suggestion
import ChannelSubscriber from '../components/channel/ChannelSubscriber';
```
or do you want to be extra clear that this only publishes events, but doesn't listen?